src/index.js

```diff
@@ -1,7 +1,49 @@
 import './style.css';
-import displaySection from './display';
-import formSection from './form';
 
-const main = document.querySelector('.main');
+const displayGame = document.querySelector('.display');
+const form = document.querySelector('.form');
+const username = document.querySelector('.username');
+const userscore = document.querySelector('.userscore');
+const refreshBtn = document.querySelector('.refresh');
 
-main.append(displaySection, formSection);
\ No newline at end of file
+async function postData(gamerName, gamerScore) {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/ygWEfybemzwhtBzH65xA/scores/', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user: gamerName,
+      score: gamerScore,
+    }),
+  })
+    .then((res) => res.json());
+  return response;
+}
+
+form.addEventListener('submit', (e) => {
+  const usernameValue = username.value;
+  const userscoreValue = userscore.value;
+  e.preventDefault();
+  postData(usernameValue, userscoreValue);
+  form.reset();
+});
+
+async function getData() {
+  const refreshData = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/ygWEfybemzwhtBzH65xA/scores/');
+  const refreshJsonData = await refreshData.json();
+  return refreshJsonData;
+}
```

Comment:
- [ ] Please change these to es6 arrow function syntax. As stated in the project requirement [particularly here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.) in the JavaScript best practices section, it's important you use es6 syntax.